package.json

```diff
@@ -20,7 +20,8 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
-    "array-changes": "1.0.3",
+    "array-changes": "1.2.1",
+    "array-changes-async": "2.0.2",
```

Comment:
Do you think it would be possible to support the sync and the async cases from one library?